src/works/chapter-getters.ts

```diff
@@ -28,35 +27,29 @@ export const getChaptersList = ($chapterIndexPage: ChapterIndexPage) => {
       publishedAt: dateNode.text().replace(/[\(\)]/g, ""),
       // We rebuild the url so it gets the full path
       url: getWorkUrl({ workId, chapterId }),
-    });
-  });
-  return chapters;
+    }
+  }).get();
 };
 
 export const getWorkTitle = ($chapterIndexPage: ChapterIndexPage) => {
   return $chapterIndexPage(".works-navigate h2 a[href^='/works/']").text();
 };
 
-export const getWorkAuthors = (
-  $chapterIndexPage: ChapterIndexPage
-): Author[] => {
+export const getWorkAuthors = ($chapterIndexPage: ChapterIndexPage): Author[] => {
   const authors: Author[] = [];
   const authorNode = $chapterIndexPage(".works-navigate h2 a[rel='author']");
   if (authorNode.text().trim() === "Anonymous") {
     return [{ username: "Anonymous", pseud: "Anonymous", anonymous: true }];
   }
-
-  if (authorNode.length !== 0) {
-    authorNode.each((i, element) => {
+  return authorNode.length !== 0
+    ? authorNode.map((_, element) => {
       const url = element.attribs.href;
       const [, username, pseud] = url.match(/users\/(.+)\/pseuds\/(.+)/)!;
-
-      authors.push({
+      return {
         username: username,
         pseud: decodeURI(pseud),
         anonymous: false,
-      });
-    });
-  }
-  return authors;
+      }
+    }).get()
+    : [] as Author[];
```

Comment:
Ternary operators are useful, but they can be cognitively heavy when large. If you immediately return `[]` as soon as you know that the `authorNode` is empty, that will make the code much easier to reason about. Same in the code below.